.circleci/src/jobs/deliver_android.yml

```diff
@@ -16,39 +17,36 @@ steps:
   - restore_environment_variables
   - restore_cache:
       keys:
-        - 2-gems-{{ arch }}-{{ checksum "Gemfile.lock" }}
+        - 2-gems-{{ arch }}-{{ checksum "native/Gemfile.lock" }}
         - 2-gems-{{ arch }}-
   - run:
-      command: bundle check || bundle install --path vendor/bundle
+      command: bundle config set path 'vendor/bundle'
```

Comment:
this path didn't change?

Reply:
No, because of the `working_directory` below.